package/ios/RNCSliderComponentView.mm

```diff
@@ -152,7 +152,7 @@ - (void)updateProps:(const Props::Shared &)props oldProps:(const Props::Shared &
         }
     }
     if (oldScreenProps.disabled != newScreenProps.disabled) {
-        slider.enabled = !newScreenProps.disabled;
+        [view setDisabled: newScreenProps.disabled];
```

Comment:
```suggestion
        [slider setDisabled: newScreenProps.disabled];
```
`RNCSliderComponentView` gets compiled for new architecture which is why the CI fails for Fabric iOS.